### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by zero77**
Add a link to a lot more fingerprinting tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,4 +44,4 @@ Special thanks to:
 
 If you want to support this addon you can donate to the following addresses:
  * bitcoin: 159Y9BLcfHyrp6wj6f3syEuk92xkRVTiie
- * monero: 482QYZaagALWtPmwbptwBaexDYmcVsJrhJp2VVjTgjYA3Kk1YyMdSg9Wz2qz1Gh31E843PFVCDWS4hR4Bjf6ipWuB9iz2cs
```
Comment (kkapsner) [inline README.md]: Why did you change something here?

**Comment by kkapsner** [comment]
Thanks for the input. But actually I would rather plug https://github.com/ghacksuserjs/ghacks-user.js/wiki/Appendix-A---Test-Sites as it seems more matching with CB.

File: README.md (hunk 1 above)
Comment (zero77) [reply README.md]: No idea, i just copied and pasted it from hear.
https://github.com/pyllyukko/user.js#fingerprinting-tests

Did i move that, or add it, i am not sure.
